test/tbb/test_multifunction_node.cpp

```diff
@@ -541,11 +542,11 @@ TEST_CASE("Test ports retrurn references"){
     test_ports_return_references<tbb::flow::rejecting>();
 }
 
-//! NativeParallelFor testing with various concurrency settings
-//! \brief \ref error_guessing
-TEST_CASE("Lightweight testing"){
-    lightweight_testing::test<tbb::flow::multifunction_node>(10);
-}
+// //! NativeParallelFor testing with various concurrency settings
+// //! \brief \ref error_guessing
+// TEST_CASE("Lightweight testing"){
+//     lightweight_testing::test<tbb::flow::multifunction_node>(10);
+// }
```

Comment:
I faced some issues with this test and will investigate them as part of this patch. 
But I guess the overall review can be started before making the fix

Reply:
Fixed